package.json

```diff
@@ -24,17 +24,19 @@
     "precommit": "yarn lint && ember test"
   },
   "dependencies": {
-    "broccoli-funnel": "^1.1.0",
-    "broccoli-merge-trees": "^2.0.0",
     "ember-cli-babel": "^6.0.0-beta.3",
     "medium-editor": "^5.23.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
+    "broccoli-funnel": "^1.1.0",
```

Comment:
@willviles I am not sure if this should be `devDependecies` or `dependencies`. Is there case when ember app do not have `broccoli-merge-trees`, `broccoli-stew` or `broccoli-funnel` as dependencies? If so then we should include them as `dependencies`.